### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by ondrejmirtes**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,9 @@
     },
 
     "autoload-dev": {
-        "App\\Tests\\": "tests/"
+        "psr-4": {
+            "App\\Tests\\": "tests/"
```
Comment (ondrejmirtes) [inline composer.json:17]: Your original error message was about autoloading `DummyTest`, not `PHPUnit\Framework\TestCase`, so this was also wrong.

Comment (rask) [reply composer.json:17]: Yes, silly mistake, thanks!

File: composer.json [hunk 2]
```diff
@@ -18,7 +19,7 @@
     },
 
     "scripts": {
-        "phpstan": "tools/phpstan analyze",
+        "phpstan": "vendor/bin/phpstan analyze",
```
Comment (rask) [inline composer.json:22]: I changed this back to `tools/phpstan analyze` and it seemingly loads the PHPUnit extension just fine, at least I did get proper errors when using the framework ruleset and trying `assertSame(true, ...)`.

Comment (ondrejmirtes) [reply composer.json:22]: I changed this for a reason. Composer enforces the installation of compatible versions between PHPStan and its extensions. If you roll back this change, you'll have PHPStan installed twice for no reason.

**Pull request merged**